WhaleSpotting/react-app/src/components/ReportSighting.tsx

```diff
@@ -0,0 +1,92 @@
+import "../styles/ReportSighting.scss";
+import React, { useState } from "react";
+
+export default function ReportSighting(): JSX.Element {
+    const [datetime, setDatetime] = useState("");
+    const [location, setLocation] = useState("");
+    const [species, setSpecies] = useState("");
+    const [quantity, setQuantity] = useState("");
+    const [longitude, setLongitude] = useState("");
+    const [latitude, setLatitude] = useState("");
+    const [orcaPod, setOrcaPod] = useState("");
+    const [imageUrl, setImageUrl] = useState("");
+    const [description, setDescription] = useState("");
+
+    const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        const sighting = { datetime, location, species, quantity, longitude, latitude, orcaPod, imageUrl, description };
+        console.log(sighting);
+    };
+
+    return (
+        <div className="report-sighting" data-testid="report-sighting">
+            <img className="banner-image" src="https://ssl.tzoo-img.com/images/tzoo.1.0.685904.shutterstock_567677317.jpg" alt="whale" />
+            <div className="container">
+                <div className="title">Report Your Sighting</div>
+                <form action="#" onSubmit={(e)=>handleSubmit(e)}>
+                    <div className="card-component">
+                        <div className="sighting-details">
+                            <div className="input-box">
+                                <label className="details">Sighting Date Time</label>
+                                <input name="datetime" type="date" placeholder="Enter Sighting Date Time" required
```

Comment:
Could we just use sentence case for these placeholders :)

Reply:
fixed